### DISCUSSION TITLE
Branch to write pull request against?

### DISCUSSION THREAD

**Discussion by leaumar** [Tichau/FileConverter#91] [Q&A]
I just thought I'd give it a try to write a pull request for my issue #77 , but it seems your repo has a long-dead `master` branch and a very active `integration` branch I don't know the purpose of. If I were to write a pull request, I assume it'd best be against `integration`?

**Comment by Tichau** [comment]
Hello, sorry for the late response, the master branch reflect the state of the program that can be downloaded on the website, I've worked on some improvements and have done a lot of refactor on all the program since on the integration branch. I'm currently trying to put more time in the project to release a new version with all the improvements done over the years.

To answer your question, you're right, If you want to work on an issue, you can start from the integration branch.

**Reply by Tichau** [reply]
I've marked the integration branch as default branch so new pull request will automatically start from it.
